src/constants/index.js

```diff
@@ -3,14 +3,13 @@
 const { stripIndents } = require('common-tags');
 
 const DEFAULT_HEADING = (
-  strings,
   owner,
   repo,
 ) => `🚀 *${owner}/${repo} deployed to production!* Related stories:\n _PRs need correct syntax to appear in the list. Problem? Please report it <https://github.com/target/captains-log/issues|here>._
 `;
 
-const EMPTY_MESSAGE = stripIndents`\n
-  *No stories linked in this release.* 🤷‍♀️ \n
+const EMPTY_MESSAGE = (owner, repo) => stripIndents`\n
```

Comment:
I would go with `({ owner, repo })` since we may wish to add more fields to the message in the future, and I find named parameters to be more readable, easier to offer optional defaults for each, etc. Good reading: https://ariya.io/2011/08/hall-of-api-shame-boolean-trap.